src/pow_avx.c

```diff
@@ -681,6 +681,22 @@ static int8_t *PoWAVX_getPoWResult(void *pow_ctx)
     return ret;
 }
 
+static uint64_t PoWAVX_getHashTimes(void *pow_ctx)
```

Comment:
There are various lines in common among `PoWAVX_getHashTimes`, `PoWC_getHashTimes`, etc. I don't think we really need that.

Reply:
@marktwtn You can calculate the `HashTimes` inside the end of `doThePoW` function and store it. Then we can just use the interface get the `HashTimes` rather than duplicate `getHashTimes` functions in each implementation.

```
uint64_t getHashTimes(void *pow_ctx)
{
    return pow_ctx->hashtimes;
}
```

Reply:
But the code would look like this
```
uint64_t getHashTimes(void *pow_ctx)
{
    return ((PoW_SSE_Context *) pow_ctx)->hashtimes;
}
```
and can not be used in other implementation.

I guess the `getHashTimes` function should remain the same and change the `PoW[Impl]_getHashTimes` function like this:

```
static uint64_t PoW[Impl]_getHashTimes(void *pow_ctx)
{
    return ((PoW_[Impl]_Context *) pow_ctx)->hashtimes;
}
```

Reply:
You are right.